Core/PrivatTransactionsImporter.cs

```diff
@@ -60,9 +60,12 @@ private static async Task<List<Transaction>> LoadData(Date date, Credentials cre
 
         private static string ByteArrayToString(byte[] ba)
         {
-            StringBuilder hex = new StringBuilder(ba.Length * 2);
-            foreach (byte b in ba)
-                hex.AppendFormat("{0:x2}", b);
+            var hex = new StringBuilder(ba.Length * 2);
+            foreach (var b in ba)
+            {
+                _ = hex.AppendFormat("{0:x2}", b);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4931949)